src/tests/asyncTests.ts

```diff
@@ -150,14 +201,20 @@ export class AsyncTests {
         return null;
       }
 
-      return await this.formatAsyncResults(
+      const formattedResults = await this.formatAsyncResults(
         { queueItem, runId: testRunId },
         getCurrentTime(),
         codeCoverage,
         runResult.testRunSummary
       );
+
+      await this.writeResultsToFile(formattedResults, testRunId);
```

Comment:
What are the following actions after writing results to files?

Reply:
If I understand corectly, there was no such a feature that we will write results to file in `reportAsyncResults`, right? Then in the call of the new function `writeResultsToFile`, the side effect is that we have a new file here `path.join(os.tmpdir(), runId, 'rawResults.json')`, Why do we need the rawResults.json file?

Reply:
I add the new function in so that we could capture the data as collected from the queries, etc., but only when running with debug enable. This way we can ask customer to enable debug, run the test and the results that feed reporting will be captured. The customer can then send us the file. With it we can the "replay" reporting phase w/o access to an org.